### DISCUSSION TITLE
Option to copy the other tags/keywords as a prompt as well

### DISCUSSION THREAD

**Discussion by Donatello-za** [SupaGruen/SD-CheatSheet#10] [Ideas]
Thanks so much for this project, it is an incredibly valuable resource!

I'm wondering whether it be possible to have a secondary prompt that contains the style prompt along with the other details of that style. For example, if I look at the ADAMS, ART (ARTHUR) style, the prompt is:
style of Arthur Adams.

What if we could have a secondary box below that one that contains something like:
style of Arthur Adams, Illustration, Comic, DC Comics, Marvel, Cover Art, B&W, USA

I can see that the length of the prompt will be a problem with your layout though and one would need to find a workaround for it. If I have some time I can perhaps make the changes and do a Pull Request if you think something like this would be valuable?

**Comment by SupaGruen** [comment]
Bare with me, I'm not sure why one would need to copy these keywords?
They are just what I used as categories to make it easier to search for. They are either already in the generated image or just there to help place that artist.

**Reply by Donatello-za** [reply]
> Bare with me

Thanks for responding to my post. :) I hope I'm not missing something but how I've been using your cheatsheet is this:

I use Automatic1111 together with img2img. I basically start with using a base image that may not have the same style I wish to get from the output, it may even be a photo (so no metadata). If I only copy the prompt "style of Richard Anderson" and use it as is I don't get the quite get the results I want but if I also add some of the tags as keywords it helps quite a lot.

For example, lets take "style of Richard Anderson" together with some basic details of what I want.:
style of Richard Anderson, man in armor, intricate, elegant, highly detailed, centered

This tend to give me an image which leans to the realistic style (using Deliberate 2.0). Now if I also add some of the tags to the prompt (removing Deliberate 2.0 and UK from it first) I tend to get a better representation of the artists artwork, i.e:
style of Richard Anderson, Illustration, Concept Art, Game Art, man in armor, intricate, elegant, highly detailed, centered

I hope what I'm saying makes sense :/

**Reply by SupaGruen** [reply]
I see, thanks for explaining.
Maybe the ['only data' view](https://supagruen.github.io/StableDiffusion-CheatSheet/only-data.html) works better in this case?

**Reply by Donatello-za** [reply]
> I see, thanks for explaining. Maybe the ['only data' view](https://supagruen.github.io/StableDiffusion-CheatSheet/only-data.html) works better in this case?

Yep, that is what I've been doing. I basically have two tabs running, one with Cheat Sheet and one with the data only. So it is no mission critical issue, just a suggestion. :) Thanks for discussing it at least, it isn't a problem if you can't fit this in, we all have busy lives.

**Reply by SupaGruen** [reply]
I added a checkbox (you can find on the about page) to turn the copying of the categories on.

**Reply by Donatello-za** [reply]
> I added a checkbox (you can find on the about page) to turn the copying of the categories on.

That is awesome, it works like a charm! Thanks so much!
